parcels/particleset.py

```diff
@@ -1037,18 +1052,19 @@ def execute(
 
             time_at_startofloop = time
 
-            next_input = self.fieldset.computeTimeChunk(time, dt)
+            #next_input = self.fieldset.computeTimeChunk(time, dt)
 
             # Define next_time (the timestamp when the execution needs to be handed back to python)
-            if dt > 0:
-                next_time = min(next_prelease, next_input, next_output, next_callback, endtime)
-            else:
-                next_time = max(next_prelease, next_input, next_output, next_callback, endtime)
+            #if dt > 0:
+            #    next_time = min(next_prelease, next_input, next_output, next_callback, endtime)
+            #else:
+            #    next_time = max(next_prelease, next_input, next_output, next_callback, endtime)
 
+            next_time = endtime
```

Comment:
This was hard-coded, just to get things working for the simple case with one time level in the UXArray dataset.